src/main/resources/edu/cornell/kfs/rass/cu-spring-rass.xml

```diff
@@ -58,11 +58,35 @@
 		<property name="moduleConfiguration" ref="cu-rass" />
 	</bean>
 
-	<bean id="rassService" class="edu.cornell.kfs.rass.batch.service.impl.RassServiceImpl">
-		<property name="rassFilePath">
-			<value>"${staging.directory}/rass"</value>
-		</property>
-	</bean>
+	<bean id="rassService" parent="rassService-parentBean"/>
+	<bean id="rassService-parentBean" abstract="true" class="edu.cornell.kfs.rass.batch.service.impl.RassServiceImpl"
+			p:rassFilePath="${staging.directory}/rass"
+			p:batchInputFileService-ref="batchInputFileService"
+			p:batchInputFileType-ref="rassXmlInputFileType"
+			p:fileStorageService-ref="fileStorageService"
+			p:rassRoutingService-ref="rassRoutingService"
+			p:dataDictionaryService-ref="dataDictionaryService"
+			p:routeHeaderService-ref="enDocumentRouteHeaderService"
+			p:agencyDefinition-ref="agencyDefinition"
+			p:documentStatusCheckDelayMillis="1000"
+			p:maxStatusCheckAttempts="60"/>
+
+	<bean id="rassRoutingService" parent="rassRoutingService-parentBean"/>
+	<bean id="rassRoutingService-parentBean" abstract="true" class="edu.cornell.kfs.rass.batch.service.impl.RassRoutingServiceImpl"
+			p:maintenanceDocumentService-ref="maintenanceDocumentService"
+			p:documentService-ref="documentService"
+			p:dataDictionaryService-ref="dataDictionaryService"/>
+
+	<bean id="rassXmlInputFileType" parent="rassXmlInputFileType-parentBean"/>
+	<bean id="rassXmlInputFileType-parentBean" abstract="true" class="edu.cornell.kfs.sys.batch.JAXBXmlBatchInputFileTypeBase"
+			p:dateTimeService-ref="dateTimeService"
+			p:cuMarshalService-ref="cuMarshalService"
+			p:directoryPath="${staging.directory}/rass"
+			p:fileExtension="xml"
+			p:fileTypeIdentifier="rassXmlInputFileType"
+			p:titleKey="message.batchUpload.title.accountingXmlDocument"
```

Comment:
What does this reference?

Reply:
Good catch, I had not updated this title key or added a value for it yet. I'll see if I can add that to the new RASS properties file that was noted in the other comment.